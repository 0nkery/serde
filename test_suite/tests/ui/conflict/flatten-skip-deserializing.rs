#[macro_use]
extern crate serde_derive;

#[derive(Deserialize)]
struct Foo {
    #[serde(flatten, skip_deserializing)]
    other: Other,
}

#[derive(Deserialize)]
struct Other {
    x: u32,
}

fn main() {}
