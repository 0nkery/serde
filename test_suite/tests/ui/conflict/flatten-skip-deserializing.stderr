error: #[serde(flatten] can not be combined with #[serde(skip_deserializing)]
  --> $DIR/flatten-skip-deserializing.rs:12:10
   |
12 | #[derive(Deserialize)]
   |          ^^^^^^^^^^^

error: aborting due to previous error

